### ISSUE TITLE
Add assignment3.html

### ISSUE THREAD

**Issue by HarrisonJackson** [smu-gui-spring-2015/ngilbert#10]
Hey can you add the html for assignment 3 landing page?

If your index page was supposed to be that, there is no clear call to action

**Comment by nigelg11861** [comment]
Yes, that was supposed to be the index.  I will modify it real quick to make sure the call to action is more explicit.
